lms/djangoapps/courseware/management/commands/tests/test_dump_course.py

```diff
@@ -205,15 +204,6 @@ def check_export_file(self, tar_file):  # pylint: disable=missing-docstring
         assert_in('edX-simple-2012_Fall/sequential/Lecture_2.xml', names)
 
 
-class CommandsXMLTestCase(CommandsTestBase):
```

Comment:
I've got an outstanding PR here: 
https://github.com/edx/edx-platform/pull/11042
that removes the usage of the XMLModuleStore in tests. Feel free to raid that for any fixes that might be helpful. But I think it's fine to remove this class.